code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -16,6 +16,7 @@
 	var/alien_disarm_damage = 30 //Aliens deal a good amount of stamina damage on disarm intent
 	var/alien_slash_damage = 20 //Aliens deal a good amount of damage on harm intent
 	var/alien_movement_delay = 0 //This can be + or -, how fast an alien moves
+	var/queen_strength = 0
```

Comment:
instead of making your own checks with this to pry open doors, I wonder if you could use `TRAIT_FORCE_DOORS` instead.

Reply:
Works via said trait now 👍